api/candlepin-api-spec.yaml

```diff
@@ -1343,6 +1517,90 @@ components:
             upstreamConsumer:
               $ref: '#/components/schemas/UpstreamConsumerDTO'
 
+    ProductContentDTO:
+      description: Product content mapping exposed to the API
+      required:
+        - id
+      properties:
+        content:
+          $ref: "#/components/schemas/ContentDTO"
+        enabled:
+          type: boolean
+
+    ProductLegacyDTO:
+      description: DTO representing the product data exposed to the API
+      required:
+        - id
+      allOf:
+        - $ref: '#/components/schemas/TimestampedEntity'
+        - type: object
+          properties:
+            id:
+              type: string
+              example: 5051
+            uuid:
+              type: string
+              example: ff808081554a3e4101554a3e9033005d
+            name:
+              type: string
+              example: Admin OS Developer Bits
+            multiplier:
+              type: integer
+              format: int64
+              example: 1
+            attributes:
+              type: array
+              items:
+                type: object
+                additionalProperties:
+                  type: string
```

Comment:
Why does this not just use AttributesDTO instead?

Reply:
Done. *LegacyDTO dtos were removed